modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -374,9 +392,11 @@ protected function actions(array $form, FormStateInterface $form_state) {
     // Hide the actions element if it has no buttons.
     $actions['#access'] = isset($actions['previous']) || isset($actions['next']);
     // Once these two steps are reached, the user can't go back.
-    if (in_array($this->stepId, ['offsite_payment', 'complete'])) {
+    if ($this->stepId == 'complete') {
```

Comment:
Why are we removing this logic? It's a part of the old 1.x logic, once payment is reached, the order is no longer a cart, and the customer cannot modify it anymore.